src/helpers/createBin.ts

```diff
@@ -22,12 +21,19 @@ export async function createBin({ code, language, lifeTime, maxUsage }: BinOptio
 				lifetime: lifeTime || 0,
 				maxusage: maxUsage ?? 0,
 			},
+			timeout: Number(process.env.MAX_TIMEOUT_MS),
+			retry: {
```

Comment:
to avoid duplicating code, i suggest you to make a file that export a extended got (see `got.extend()`).
Also, can you keep defaults `methods` ?

Reply:
Non, on peut pas garder les méthodes par défaut, surtout pas ici, parce que dans les méthodes par défaut il n'y a pas POST

Reply:
dans tous les cas ça change pas
et la c'est le duplication inutile, exporte une instance de got dans un autre fichier utils